docs/membership-policy.md

```diff
@@ -0,0 +1,29 @@
+# CI maintainers
```

Comment:
This policy is inspired by k8s's and k6t's. Please link to the originals

Reply:
Yep, added links to both

Reply:
I'm no github expert, but can you "Resolve" this?